sig-etcd/README.md

```diff
@@ -31,6 +31,7 @@ The [charter](charter.md) defines the scope and governance of the etcd Special I
 ### Chairs
 The Chairs of the SIG run operations and processes governing the SIG.
 
+* Ivan Valdes (**[@ivanvc](https://github.com/ivanvc)**), Aki Technologies
```

Comment:
@jmhbnz, could you update my company affiliation to "Inmar Intelligence"? That's the parent company. I updated my GitHub profile to reflect this. Thanks.

Reply:
Absolutely, done!

Reply:
Thank you! :)